### DISCUSSION TITLE
Help with "Your Team encountered an unknown problem" error

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#7207] [Help] [answered]
Hey everyone. Thank you for your patience while we work to resolve a difficult problem causing some people to see the error message, "Your Team encountered an unknown problem. Please reach out to our support team for details."

We have programmatically solved the problem for as many teams as possible so most are now able to deploy without error. However, there may still be some teams that need us to manually solve the problem. 

The discussion in https://github.com/vercel/community/discussions/6027 has gotten large enough that it is difficult to follow. If you still see this error, please let us know here so we can fix it for you. I want to be sure no one's team was overlooked.

**Comment by codecret** [comment]
hello , please help me to fix the error my id is xJiBgD4wMqExdvDDPaj00uwz

**Reply by amyegan** [reply]
Thanks for reaching out. You should be able to deploy now.

**Comment by Nutshell-King** [comment]
please help me to fix the error
Vercel ID: XOqRvlRf7PaZlJuUYORdVegb

**Reply by swarnava** [reply]
You should be unblocked now!

**Comment by HorthmanOdame** [comment]
Hello @amyegan sorry to bother, but I'm currently facing the same problem. Here is my ID: rwzMDApapA43LwgnrthNuAbz.
Thank you for your time, and have a great day.
Kind regards

**Reply by swarnava** [reply]
You should be unblocked now!

**Reply by HorthmanOdame** [reply]
Thanks @swarnava . Have a nice day!

**Comment by walterlar** [comment]
please help me to fix the error
Vercel ID: 2jGCnmo6UunChuic061wJRDC

**Reply by amyegan** [reply]
It should work for you know

**Comment by YuAnReL** [comment]
please help me
Vercel ID:PobclS2x5EDOrr9uAE8eCUn1

**Reply by amyegan** [reply]
You should be able to deploy now

**Comment by CocDap** [comment]
please help me :
My vercel id : JbFTTcBY6pczkPODfwd4EgND

**Reply by amyegan** [reply]
It should work for you now

**Reply by YuAnReL** [reply]
Thanks, Have a nice day!

**Comment by LeoShumacher** [comment]
I can't import my project, I don't know how to solve it, please help me
![Screenshot_185](https://github.com/vercel/community/assets/123395012/618f7b4f-06a4-4402-b815-c581cb49cc51)

**Reply by amyegan** [reply]
You should be able to deploy now

**Reply by LeoShumacher** [reply]
Now everything is fine here, thank you

**Comment by vustinchdev** [comment]
hello , please help me to fix the error Your Team encountered an unknown problem. Please reach out to our support team for details.

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by xuxang6** [comment]
please help me deploy , I got error "Your Team encountered an unknown problem. Please reach out to our support team for details"

**Reply by amyegan** [reply]
You should be able to deploy now 🙂

**Comment by akahlouts** [comment]
Please help me as well

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by dhruvpatel374** [comment]


**Reply by amyegan** [reply]
You should be able to deploy now 🙂

**Comment by viruscoding** [comment]
please help me :
My vercel id : syj5lyD6RTiO0wHJtC1dnpLn

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by A-Faruki** [comment]
Please help me
My vercel id: UNC1ApfPq9gtnpccNSOLzbsO

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by ghost** [comment]
I've got the same error
My vercel id: h2w3rQjbJ6BKb3hC9SFsg2Fz

**Reply by amyegan** [reply]
You should be able to deploy now 🙂

**Comment by phantom930** [comment]
I faced also same issue right now.
My vercel id: gKOjTje7B4ylggjffWZ6skaW

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by sanketof** [comment]
I also have the same issue.
Vercel ID: fyF6ZiFY46gJjFUljEMuezSF

**Reply by amyegan** [reply]
You should be able to deploy 🙂

**Comment by bongende** [comment]
Hi!
please im still getting this error 
here is my ID : YF8P1SYNojM83qhV9fv8N7wl

**Reply by swarnava** [reply]
It should work for you now 🙂

**Comment by RahulRazj** [comment]
Hi, Amy.
I have the same error. Please take a look.
My Vercel Id: AV8XZcoA9WIH2PDakwTBLYjJ

**Reply by swarnava** [reply]
It should work for you now 🙂

**Comment by Ohriel94** [comment]
Hi, I have the same issue.
Here is my ID : AFGA23ve67YdD3pwtkN2beck

**Reply by swarnava** [reply]
It should work for you now 🙂

**Reply by Ohriel94** [reply]
Thank you

**Comment by itsodepth** [comment]
Help me, i have same issue :')
Here is my Vercel ID : PNtPwmHsE0ayJSAVOyDIWEIO

**Reply by amyegan** [reply]
It should work for you now

**Reply by itsodepth** [reply]
Thank you!

**Comment by skyv** [comment]
Hi Amy, I've got the same error. Please help:
My vercel id: GYf1UTrAdhJ5AEJMaN9o7Nlz

**Reply by swarnava** [reply]
It should work for you now 🙂

**Reply by skyv** [reply]
Thank you 👍

**Comment by maxharper26** [comment]
Hey I am experiencing it too.
Vercel id: 98GOTKDQ1UZ1K7ts1nhAxSAf

**Reply by swarnava** [reply]
It should work for you now 🙂

**Comment by lrangell** [comment]
Same
Vercel id: BDVH3zSTIvNX4C67WKT8AeLi

**Reply by swarnava** [reply]
It should work for you now 🙂

**Comment by q1134248919** [comment]
Same
Vercel id: UKZgdoXPxrJBo7Pm7GBVCDdM

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by aplombDev** [comment]
![Uploading Screenshot 2024-06-25 175322.png…]()

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by arekyao** [comment]
Same Issue.
My Vercel id: NJNcjjMaGRYQtQzrlFiu9HGm
Thanks

**Reply by amyegan** [reply]
It should work for you now 🙂

**Reply by arekyao** [reply]
It works now, thanks

**Comment by caiomateus-dev** [comment]
Can you help me?
Same issue.

Vercel ID: dPyJP8tVujcqbiHeQm5QEsyW

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by stepandev8** [comment]
Can you help me?
Same issue.

Vercel ID: 7IALltjeCSSKFOoGPSkxn3VB

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by bhargav462** [comment]
Can you help me?
Same issue.

Vercel ID: 6i4f9ieFtj47PzooLYvBFHqb

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by zhhltc** [comment]
I'm facing same issue. please help me, my vercel ID:gP98eg3kFo370cxLY1cdj3Oz

**Reply by amyegan** [reply]
It should work for you now 🙂

**Reply by zhhltc** [reply]
Thanks, Have a nice day!

**Comment by user686** [comment]
please help me
Vercel ID:mKGVwmLyGCnzXy9TixJ2R7C0

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by moayedalhagy** [comment]
Hello, help me please !
my id : 6GbAeCHYJqvSK1Tbg9nM0doW

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by mertcakdogan** [comment]
Hi, please help me !

Vercel ID: f1GjGUiX0hlwigF4nLQWTwF8

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by thedoggybrad** [comment]
I encounter the same issue. Wanted to deploy a new Github-imported project but I receive the same error that my teams encounter an error.

Here's my Vercel ID for your reference: 86OecwSAxRVMe7VdREVNFhqF

**Reply by amyegan** [reply]
It should work for you now 🙂

**Reply by thedoggybrad** [reply]
> It should work for you now 🙂 

Thank You! It's fine now.

**Comment by Vanush27** [comment]
@amyegan

Hi. Please help me

Vercel ID:  zCXDmxG3ApKRgG3BDtneqLrI

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by arturgeghamyan** [comment]
Can you help me?
Same issue.

Vercel ID: 2N0QNEAJNsjhDDfcNGDQYi3w

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by nguptaesaral** [comment]
same issue
team_id: team_CVwkWPYQhDG0MqsTvYfLmJLR

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by ghost** [comment]
Hi. I have same issue, Would you mind helping me out?

Vercel ID: B8E7jbvUJHSnwZdPTB1VTZv9

**Reply by amyegan** [reply]
It should work for you now 🙂

**Reply by ghost** [reply]
> It should work for you now 🙂

Thank you! @amyegan ,Have a nice day!

**Comment by Durgeshpratapsingh123** [comment]
please help me to fix the error
Vercel ID: v3SgqNlebfUp4EwKckCgYTqg

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by kaylangenberg** [comment]
Hi, I have the same issue. Please fix. 

Vercel ID: SWmQGIBn4cnzGNBlLvlelyu6

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by Murat-Sahin1** [comment]
I experience the same problem, could you please resolve it?
My Vercel ID: hEvshYMgClQTjy8n2k07SK4C

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by webwiseks** [comment]
I experience the same problem, could you please resolve it?
My Vercel ID: eQ0VPB7Nx34CsRRvANZ9RDRM

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by zayn-0101** [comment]
Same problem, please help me. My Vercel ID is: aJkwMxDtnDeJDRhT2xCJug5i
Best regards, have a great day!!

**Reply by amyegan** [reply]
It should work for you now 🙂

**Reply by zayn-0101** [reply]
Thanks so much

**Comment by JoHena** [comment]
Need help deploying. ID: Fn6RhHLut2nNFQQc0XUkjiY5 
Thanks

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by immafla** [comment]
Hi, please help me! 

my ID: CFWWv8ihJFZSCYzqkwG06ra7

Thanks so much

**Reply by pawlean** [reply]
@immafla Should work now, let us know if you still see this error.

**Reply by immafla** [reply]
> @immaflaDebería funcionar ahora, avísenos si aún ve este error.

Its Working, thankyou so much!!!

**Comment by tmanas06** [comment]
Hello there ,I experience the same problem " Your Team encountered an unknown problem. Please reach out to our support team for details.  " , could you please resolve it?
ID: MNlMGFyJ39XrLPoF6Ysxsilc

**Reply by pawlean** [reply]
@tmanas06 Should work now, let us know if you still see this error.

**Reply by tmanas06** [reply]
Its working , thankyou so much

**Comment by Kareemtaiye** [comment]
same issue here ma'am;
vercelID : zMIHeJg8GNIdoHtRFrdgW8de

**Reply by pawlean** [reply]
@Karimtaiye Can you re-share your ID? I don't think this is correct.

**Comment by appdevprime** [comment]
I just experienced this now

**Reply by pawlean** [reply]
@appdevprime Can you share your ID?

**Comment by Ahmad-Rabani** [comment]
can you help me please?
i am facing the same problem.
vercal id: 0pwBMjWKFonQHg2mo29Am57c

**Reply by pawlean** [reply]
@Ahmad-Rabani Should work now, let us know if you still see this error.

**Reply by Ahmad-Rabani** [reply]
Thank you soo much it worked really appreciated.

**Comment by rilindkycyku** [comment]
I have the same problem. My id is wtaBVlpvyTR8G4kQI8b37xBg

**Reply by pawlean** [reply]
@rkycyku Should work now, let us know if you still see this error.

**Comment by ayushmaan32** [comment]
facing same error my id is Qii7CXfHwvCahzymMUm0cDDq

**Reply by pawlean** [reply]
@ayushmaan32 Should work now, let us know if you still see this error.

**Comment by NathanSchoepps** [comment]
I have the same error.. my id is : ZjGaqAyBn4xYlG7S5wNgAeQn

**Reply by amyegan** [reply]
@NathanSchoepps It should work for you now 🙂

**Reply by NathanSchoepps** [reply]
Thank you :)

**Comment by tengweiorg** [comment]
hello , please help me to fix the error my id is kIWzvZGWdB0lkEtwrfJgM6Ut , thanks.

**Reply by pawlean** [reply]
@tengweiorg Can you check now? Should be working!

**Comment by yeqiling** [comment]
can you help me please?
i am facing the same problem.
my id is : BzsaaxJh4CM7VP0RYI6XnSil 
Team id is: team_STb5rho7ymHk3rd1tOZ1khNY
thanks.

**Reply by pawlean** [reply]
@yeqiling Should work now!

**Comment by Pipoteex** [comment]


**Reply by Pipoteex** [reply]
I have the same problem!

**Reply by Pipoteex** [reply]
With the same problem

**Reply by pawlean** [reply]
@Pipoteex Can you share your team or user ID?

**Comment by soda1107** [comment]
hello, please help me to fix the error my id is SQulVdMdOHOYrkMjfhP748Wa. thanks!

**Reply by pawlean** [reply]
@soda1107 You should be good now!

**Reply by soda1107** [reply]
thanks!

**Comment by mdbailin** [comment]
Saw the unknown problem screen of death this morning. Here's my team ID: team_6o5Av8VwpXukwDXyv9uJLzfJ Please go do the thing. Thank you!

**Reply by pawlean** [reply]
@mdbailin Did the thing. You should be good now 😄

**Comment by iconofsin** [comment]
zJWHR1cHruDG5V14syitt6XI

"Your Team encountered an unknown problem. Please reach out to our support team for details"

**Reply by pawlean** [reply]
@iconofsin This should work now!

**Comment by madushas** [comment]
same issue.
Vercel ID: ygBLuWcXTEyA8NXyET5QcllZ

**Reply by pawlean** [reply]
This should be fixed now!

**Comment by gohil10** [comment]
Vercel ID: UT7lIqyNJNGNaEeVmKqhVxsM

Issue: "Your Team encountered an unknown problem. Please reach out to our support team for details"

**Reply by pawlean** [reply]
This should work now!

**Reply by gohil10** [reply]
Thanks!

**Comment by anggaaasetiyawan** [comment]
vercel id: 9qaLDgBpfIDXcceII9diovLG
Issue: "Your Team encountered an unknown problem. Please reach out to our support team for details"

**Reply by pawlean** [reply]
@foxxw0lf You should be ready now :)

**Comment by TimCookXXX** [comment]
Vercel ID: h3QCTg6OGk7kn7S1iaMWnMni
Your Team encountered an unknown problem. Please reach out to our support team for details.
Help me:)

**Reply by pawlean** [reply]
@TimCookXXX You should be ready now :)

**Comment by NameGoesThere** [comment]
VaWPfYxC5Hqoc2KYvQquiK7T
please

**Reply by pawlean** [reply]
Hi @NameGoesThere! This should now work.

**Comment by willnguyen23303** [comment]
Vercel ID: l9koqv7Trs1SaVQM68a9jop7
Issue: Your Team encountered an unknown problem. Please reach out to our support team for details. [Learn More](https://vercel.com/help)
Please help me :)

**Reply by pawlean** [reply]
Fixed!

**Reply by willnguyen23303** [reply]
Thanks for your help :)

**Comment by FurkanMuslu** [comment]
Vercel ID: cFLbxnVDjgY8nh9SCdeQd6ZN
Issue: "Your Team encountered an unknown problem. Please reach out to our support team for details"

**Reply by pawlean** [reply]
This should be working for you now!

**Comment by AgilamOrganization** [comment]
team_k4k0qHsCURuythPL4DLFXbpK
Issue: "Your Team encountered an unknown problem. Please reach out to our support team for details"
Please help me :)

**Reply by pawlean** [reply]
This should be working for you now!

**Comment by eskanusa-booking** [comment]
Help fix the error
![image](https://github.com/vercel/community/assets/174825393/820488a9-6a1b-4257-9ba9-1817becb374a)
Vercel ID: bAaLdlrVoUlH2fcGeqUvX625
Team ID: team_vQPTAn0DVVDmeu7zcESkTk5g

Thanks in advance

**Reply by pawlean** [reply]
This should be working for you now!

**Comment by phunsok** [comment]
Your Team encountered an unknown problem. Please reach out to our support team for details. [Learn More](https://vercel.com/help)

Vercel ID: 9KoTOh6ckFgzD9pn9PGbG5MO

**Reply by pawlean** [reply]
This should be working for you now!

**Comment by adrianmarkperea** [comment]
Facing the same issue, kindly asking for help

Vercel ID: uR4kGYFNlI3hzciIH201dmj9

**Reply by pawlean** [reply]
This should be working for you now!

**Comment by beatrizcroucas** [comment]
Hello, could you help me please?
my id: team_quWbd4vAJ94lexsQdi2cevn6

**Reply by pawlean** [reply]
This should be working for you now!

**Reply by beatrizcroucas** [reply]
thank you so much!

**Comment by KeshavManuja** [comment]
HI, I am learning nextjs from documentation, stucked at unable to deploy, id is fpIsnNTUKFg40lvIcUGLHc45

**Reply by pawlean** [reply]
This should be working for you now!

**Comment by eskanusa-booking** [comment]
Hi, could you please help me with this Vercel ID: bAaLdlrVoUlH2fcGeqUvX625 and Team ID: team_vQPTAn0DVVDmeu7zcESkTk5g
Thanks

**Reply by pawlean** [reply]
This should be working for you now!

**Comment by cunhafelipe** [comment]
olá, por favor me ajude a corrigir o erro meu id é AItbEW6Lk8wQdAVzuNHN3KTd

**Reply by pawlean** [reply]
This should work now

**Comment by Mico1613** [comment]
Hello. Help me please with this problem. My id is BNroy26FMQxgPwzGbaxUNO84

**Reply by pawlean** [reply]
This should work now

**Reply by Mico1613** [reply]
Thank you. It works now

**Comment by Shiguang-coding** [comment]
Hello, could you help me please?
my Vercel ID : 6NngeIh5bRBt7CGN3I8NfxPk

**Reply by pawlean** [reply]
This should work now

**Reply by Shiguang-coding** [reply]
Thank you. It works now

**Comment by jisshub** [comment]
Error: Your Team encountered an unknown problem. Please reach out to our support team for details.
Learn More: https://vercel.com/help

Hello, could you please help me

my vercel id: Bef6vpH6zpeVAW9N3gN9iU1x

**Reply by pawlean** [reply]
This should work now

**Comment by wurasimi** [comment]
help me fix this error thank you

**Reply by pawlean** [reply]
@wurasimi What's your ID?

**Comment by wurasimi** [comment]
help me fix this issue of "Your Team encountered an unknown problem. Please reach out to our support team for details. [Learn More](https://vercel.com/help)" whilre deploying, thanks

**Reply by pawlean** [reply]
Hi @wurasimi, can you share your vercel ID or team ID?

**Comment by tuchg** [comment]
Error: Your Team encountered an unknown problem. Please reach out to our support team for details.

Hello, could you please help me

my vercel id: 1jy9rJBs1p63CChEiJQIV5Vs

**Reply by pawlean** [reply]
This should work now!

**Comment by code-mena** [comment]
I'm facing the same error just now, kindly asking for help.

My vercel id: tXhTbfNg8D2XYMbnddGgFJHG

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by ovsaltuk** [comment]
Hi, I have the same problem.
My vercel id: dn7KYThyTA4JdxxK6g2FkAKv
Team id: team_iGVRAEJUtJfWoWvu6Cm35w0K

**Reply by pawlean** [reply]
@ovsaltuk This should work now.

**Reply by ovsaltuk** [reply]
Thank you so much have a nice day

**Comment by ninynice** [comment]
Hi, I have the same problem.
My vercel id:aQsYcUzpXp2rZFTnfpO03MXA

**Reply by pawlean** [reply]
This should work for you now.

**Comment by GBMilady** [comment]
Same issue

Vercel id: B407x2k1gq8nupujXcvWKfiY
team_ye3PdCM7EWsdxqLtV4wugVRu

**Reply by pawlean** [reply]
This should be working for you now 😄

**Comment by maehey** [comment]
Hi, same problem
ID: uiw236d9VVBuY1fdIUMHIAPE

**Reply by amyegan** [reply]
It should work for you now 🙂

**Comment by uttarachatterjee** [comment]
Please help me fix the error. Vercel ID : kKTSkiELmpSqf3daEhYOUBD5
teams ID : team_s1S3P8SeR5Wsm3d1blEwg8Kj

**Reply by swarnava** [reply]
This should be fixed now!

**Reply by uttarachatterjee** [reply]
Thanks!!

**Comment by huyphamnhat-agilityio** [comment]
I got the same error. My Vercel ID: WQYTdgeUapn1kxCCjrZTkDGm. My Team ID: WQYTdgeUapn1kxCCjrZTkDGm.

**Reply by pawlean** [reply]
This should work for you now!

**Comment by globaloma** [comment]
Hello @amyegan sorry to bother, but I'm currently facing the same problem. Here is my ID: XuTzUAm3UNcM1s9G6b8ZIjQZ
and my Team ID: team_oSwd2pYUJnhP2JFXuv8nC2TJ Thank you for your time, and have a great day.
Kind regards

**Reply by pawlean** [reply]
This should work now

**Reply by globaloma** [reply]
Thanks a bunch!

**Comment by rochimups** [comment]
Hello @amyegan sorry to bother, but I'm currently facing the same problem. Here is my ID: eAeXSX7ZpAFvxlkactaIfPPw. Thank you for your time, and have a great day.

**Reply by pawlean** [reply]
This should work now!

**Comment by G87git** [comment]
Hi, please help me with the following error:

Your Team encountered an unknown problem. Please reach out to our support team for details. [Learn More](https://vercel.com/help)

**Reply by pawlean** [reply]
@G87git Can you share your team ID?

**Reply by G87git** [reply]
My team id:
team_iwVDi2zh7t7hCEmy4QMYjAIa

**Reply by pawlean** [reply]
@G87git Should work now :)

**Reply by G87git** [reply]
DANKE!

Just wanna know what happened?

**Comment by jawncodes** [comment]
Hi @amyegan / @pawlean ! Encountered the same error - here's my team ID:

```
team_JDfECRfX0kPBCUnhSGghIK3u
```

Thanks in advance!
\- jawn

**Reply by pawlean** [reply]
This should work :)

**Comment by KiritoU** [comment]
Hi, please help me too!
My vercel ID: ZLABt4LtKAWRCduSQXyKwi06

I'm importing project from Gitlab!

**Reply by pawlean** [reply]
This should work now!

**Comment by kavehlogestic-it** [comment]
i have same problem

**Reply by kavehlogestic-it** [reply]
my id is : team_QvKdpau9RrL0OnqXslIJF5Uu

**Reply by pawlean** [reply]
This should work now!

**Comment by MuhammadAhmedSuhail** [comment]
I am facing the same issue.

**Reply by MuhammadAhmedSuhail** [reply]
My team id is team_AAVnBEAuNvlUzpQk5zYvkpN1

**Reply by pawlean** [reply]
This should work :)

**Comment by akulistus** [comment]
Hello,
Having the same problem.
My Vercel ID: OK4KGEV8qyHYKmDGIPDAUCSC
My team ID: team_EAlBdk7KZrcc8srk9jdeHkDW

**Reply by pawlean** [reply]
This should work now!

**Reply by akulistus** [reply]
Thank you!

**Comment by jackypark9852** [comment]
Having the same problem: 
My ID: uyOXM0qwTUJfgFD0WHJue96s

**Reply by pawlean** [reply]
This should work now!

**Comment by stdestro** [comment]
same problem team_ntnUvB3xDxYGNYF5bLOQflps

**Reply by pawlean** [reply]
This should work now!

**Comment by mysbryce** [comment]
I got same problem my vercel id F8bcXebAtgVvnp0H8X2lzJVB

**Reply by pawlean** [reply]
This should work now!

**Comment by huaizhixu** [comment]
Having the same problem:
My ID: PJtkqaRf77RrtU0ODS7mzo6e

**Reply by pawlean** [reply]
This should work now!

**Comment by emirrozerr** [comment]
I got the same problem. "Your Team encountered an unknown problem. Please reach out to our support team for details."
Vercel ID: lnY9dmfgSy4rSz3LdcSt0QqF

**Reply by pawlean** [reply]
This should work now!

**Comment by gravitywp** [comment]
I got the same problem. "Your Team encountered an unknown problem. Please reach out to our support team for details."
Vercel ID: bkQpN7gWIPFuKJQQ8orha2Ni

**Reply by pawlean** [reply]
This should work now!

**Comment by pawlean** [comment, marked as answer]
Hi everyone! I'm closing this discussion in favour of our move to our **new community platform** https://vercel.community. 

See announcement here: https://github.com/orgs/vercel/discussions/7413

If you're seeing this issue, please head over to the new platform and start a discussion there.

**Reply by amyegan** [reply]
New thread opened at https://vercel.community/t/your-team-encountered-an-unknown-problem/173

**Comment by FarzanehHadadi** [comment]
I got the same problem. "Your Team encountered an unknown problem. Please reach out to our support team for details."
Vercel ID:NWtJCW3nYjsQMbf77KylW2aK

**Comment by realamirshad** [comment]
please help me :
My vercel id : syj5lyD6RTiO0wHJtC1dnpLn

**Comment by gentlengineer** [comment]
I've got the same error.
My vercel id: UbtKDfmjSJar9tV9lrAb1W8x

**Comment by wwrtpfa** [comment]
hello , please help me to fix the error my id，It's been 24 hours， my veecel  this is J5MBVDyhNolHTc0VAtiP8uZQ

**Comment by trupesh2408** [comment]
Hello, i'm facing the same error, this is my vercel id bMKaefTunLfErykCHtqCv3Hz

**Comment by HarutAzatyan777** [comment]
Здравствуйте, я столкнулся с той же ошибкой, это мой vercel id
wc5uMQiKbUELsLLjN9ewDpyC

**Comment by amyegan** [comment]
@FarzanehHadadi @amir-shad @luckyengineer053 @wwrtpfa @trupesh2408 @HarutAzatyan777 It should work for all of you now.

If you continue to have trouble, please reach out on [vercel.community](https://vercel.community) 🙂
